control/trapezoid_velocity_profile.go

```diff
@@ -48,11 +48,12 @@ func (s *trapezoidVelocityGenerator) Next(ctx context.Context, x []*Signal, dt t
 	var pos float64
 	var setPoint float64
 	if len(x) == 2 {
+		//nolint: exhaustive
```

Comment:
This line seems misplaced and possibly unnecessary. I _think_ it's to watch for non-exhaustive switch statements? but the next line down is a for loop, and the switch statement within the for loop is indeed exhaustive because it's got a default case. Can this be removed?